_agstate/agoric-servers/package.json

```diff
@@ -12,6 +12,6 @@
   "author": "Agoric",
   "license": "Apache-2.0",
   "dependencies": {
-    "@agoric/cosmic-swingset": "*"
+    "@agoric/cosmic-swingset": "dev"
```

Comment:
The endo changes make sense to me, I'm just a bit unclear about the `dev` part. I suppose it's because we want to keep this code up-to-date with all breaking changes? And, if we had some live version of the dapp on devnet or something, I assume we wouldn't want to use `dev`?

Reply:
This is a good question for @michaelfig. I’m doing this as a drive-by for what I believe I heard was the new vogue.

I believe the reasoning is that we’ve begun publishing dist tags to npm from every push, so these `dev` tags will keep the dapp more up-to-date. Specifically because `yarn` interprets `*` so broadly it might never update a package once it has any version at all.